Dockerfile

```diff
@@ -18,9 +18,7 @@ COPY --from=builder /app .
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
 
-RUN python manage.py migrate
-
 EXPOSE 8080
 
 # Run database migrations and start the Django application
-ENTRYPOINT ["python", "manage.py", "runserver", "0.0.0.0:8080"]
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]
```

Comment:
The `ENTRYPOINT` command specifies that the Django application should run on port 8000, but the `EXPOSE` directive specifies port 8080. Ensure that the ports match to avoid connectivity issues. You can either change the `EXPOSE` directive to 8000 or update the `runserver` command to use port 8080.